packages/documentation-site/patternfly-docs/content/developer-resources/rtl-handbook.md

```diff
@@ -11,25 +11,34 @@ The default browser behavior displays content following LTR formatting. In order
 
 ## Support in PatternFly
 
-PatternFly components have been written to use logical properties - `block-start`, `inline-end`, `block-end`, and `inline-start` - rather than `top`, `right`, `bottom`, and `left`. This allows styles like margin and padding to be correctly laid out in both LTR and RTL. However, to avoid a sweeping breaking change, PatternFly variable names still use `top`, `bottom`, `left`, and `right`.
+### Components 
 
-PatternFly components have been written to reverse directional icons that are part of the component - for example, the caret indicating expansion of various components is properly reversed. 
+PatternFly components have been written to use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) - `block-start`, `inline-end`, `block-end`, and `inline-start` - rather than `top`, `right`, `bottom`, and `left`. This allows styles like margin and padding to be correctly laid out in both LTR and RTL. However, to avoid a sweeping breaking change, PatternFly variable names still use `top`, `bottom`, `left`, and `right`.
 
-Other icons you may have in your interface will not be mirrored automatically, as it may be appropriate to mirror some but not others. Be sure to review all icons in your product and adjust appropriately. PatternFly has modifier classes listed below that can help with that.
-<br />
+**Note:** An exception to this support is with any components using popper.js, like tooltips and popovers. 
```

Comment:
Do we want to mention anything about our upgrade plan to floating-ui, or how to manage popper things in RTL for now? For example, something simple like "For now, the popper position/placement will need to be managed manually when used on pages with RTL content." cc @tlabaj @wise-king-sullyman

Reply:
Yes, I think we should mention that. I like the sentence you have here @mcoker